core/src/main/kotlin/com/exactpro/th2/read/db/core/impl/BaseQueryProvider.kt

```diff
@@ -21,14 +21,15 @@ import com.exactpro.th2.read.db.core.QueryConfiguration
 import com.exactpro.th2.read.db.core.QueryHolder
 import com.exactpro.th2.read.db.core.QueryId
 import com.exactpro.th2.read.db.core.QueryProvider
-import mu.KotlinLogging
+import io.github.oshai.kotlinlogging.KotlinLogging
 import org.apache.commons.text.StringSubstitutor
 import org.apache.commons.text.lookup.StringLookup
 import java.sql.JDBCType
 import java.sql.SQLType
 
 class BaseQueryProvider(
-    queriesConfiguration: Map<QueryId, QueryConfiguration>
+    queriesConfiguration: Map<QueryId, QueryConfiguration>,
+    queryFetchSize: Int,
```

Comment:
suggestion: should be named `defaultQuertFetchSize` I think to make it more clear what it is for. Same for the parameter in the configuration class